lib/client.js

```diff
@@ -277,6 +328,9 @@ exports.createClient = function(reference) {
             }
             debug("Request error calling %s NOT retrying!, err: %s, JSON: %s",
                   entry.name, err, err);
+            if (reportStats) {
+              reportStats(false, (err.code || err.name || 'Error') + '');
```

Comment:
To ensure conversion to string.. stupid really, but it could be something else...